splash/middleware.py

```diff
@@ -62,3 +62,13 @@ def path_matches(self, path, pattern):
                 matches = True
 
         return matches
+
+
+    def __call__(self, request):
+        """
```

Comment:
Had to upgrade the middleware again as the middleware settings require a __call__ method and the default method of mixin was calling the process_response method and raising an error if it does not exist. While we are not returning any response in case of error and returning none.